frontend/public/components/cron-job.jsx

```diff
@@ -45,9 +45,11 @@ const Details = ({obj: cronjob}) => {
           <dt>Schedule</dt>
           <dd>{cronjob.spec.schedule}</dd>
           <dt>Concurrency Policy</dt>
-          <dd>{_.get(cronjob.spec, 'concurrencyPolicy', '-')}</dd>
```

Comment:
The `_.get` calls are unnecessary, and this would already fail with a runtime error if `spec` was undefined at line 46.

Reply:
I don't think that you can specify cronjob without `spec` field